src/datachain/query/dataset.py

```diff
@@ -795,28 +794,36 @@ def apply_sql_clause(self, query: Select) -> Select:
 
 @frozen
 class SQLMutate(SQLClause):
-    args: tuple[Union[Function, ColumnElement], ...]
+    args: tuple[Label, ...]
```

Comment:
Not sure why do we use `Label` here now and why we removed `datachain.func.base.Function`.

We have a lot of these cases in tests, for example: `mutate(t1=func.rand())`.